### DISCUSSION TITLE
Serial anormal speed bps

### DISCUSSION THREAD

**Discussion by JesusPes** [SpenceKonde/ATTinyCore#684] [Support / Q & A]
Hello,
I am working with a bare ATtiny1634 chip, programmed through USBasp -Chinese- ; set the "Clock at 8 MH internal"
And when I set the speed "Serial.begin(9600);" I get a response on the PC at 1200 bps.

To get a speed of 9600 on the PC, I have to set the speed to "Serial.begin(76800);"

Could this be corrected?

**Comment by sleemanj** [comment]
Likely you have not done burn bootloader so fuses are not set and clock is therefore divided by 8 which is default for avr chips.  Do burn bootloader to set fuses.

**Reply by SpenceKonde** [reply]
Yup, this would be the problem. I have clarified this in the v2.0.0-dev documentation.

**Reply by SpenceKonde** [reply]
Yup, this is almost certainly the problem. 

This seems to be the most frequently asked question about classic AVRs, always. Even though it's like this on every classic AVR core, because these parts need fuses set to select the clock source, and we can't give the same treatment that we do on megaTinyCore for the base clock speed select because on classic AVR setting the fuse improperly will brick the chip, potentially making recovery quite difficult, and just doing "upload" should never be able to brick your chip.
